src/utils/types.ts

```diff
@@ -114,16 +120,44 @@ export type IResponseErrorValidationFault = IResponse<
   "IResponseErrorValidationFault"
 >;
 
+export const ResponseErrorValidationFault: (
+  category: FaultCategoryEnum,
+  detail: PaymentFaultEnum,
+  detail_v2: ValidationFaultEnum
+) => IResponseErrorValidationFault = (category, detail, detailV2) => {
+  const problem: ValidationFaultPaymentProblemJson = {
+    category,
+    detail,
+    detail_v2: detailV2,
+    status: HttpStatusCodeEnum.HTTP_STATUS_404 as HttpCode,
```

Comment:
maybe is related to this PR https://github.com/pagopa/io-pagopa-proxy/pull/350, but why is it 404? cc @giovanniberti